Cloudinary/Operation/02_CloudinaryMapping.md

```diff
@@ -0,0 +1,135 @@
+
+#### Prerequisites
+
+- At least one ETL attribute set mapping has been created, see [Create an ETL mapping](../../DataHub/Operation/01_ManageETLMappings.md#create-an-ETL-mapping).
+- The Cloudinary attribute has been assigned to the destination attribute set of the mapping.
+
+#### Procedure
+
+*DataHub > Settings > Tab ETL*
+
+![Attribute set mappings](../../Assets/Screenshots/DataHub/Settings/ETL/AttributeSetMappings.png "[Attribute set mappings]")
+
+> [Info] The following procedure describes exemplary how to map a Cloudinary image attribute. The procedure is largely identical to map a Cloudinary video attribute. The differences are explicitly specified at the appropriate place.
+
+1. Click the attribute set mapping you want to edit in the list of attribute set mappings.   
+  The *Mapping from source set to destination set* view is displayed.
+
+  ![Mapping](../../Assets/Screenshots/DataHub/Settings/ETL/Mapping.png "[Mapping]")
+
+2. Click the row with the Cloudinary image destination attribute.   
+  The *Settings* section is displayed on the right side.
+
+  ![Cloudinary destination attribute](../../Assets/Screenshots/DataHub/Settings/ETL/CloudinaryDestinationAttribute.png "[Cloudinary destination attribute]")
+
+  > [Info] Depending on the selected row, the fields displayed in the *Settings* section differ:
+  - If the selected row contains only a destination attribute, the destination attribute as well as the *Extension* and *Override change tracking mode* drop-down lists are displayed.
+  - If the selected row contains an attribute mapping with source, destination attribute and the extension, the destination attribute as well as the *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are displayed. The *Source attribute*, *Extension* and *Override change tracking mode* drop-down lists are preset. If the selected extension requires further configuration settings, the *Configuration* section is displayed below the *Settings* section.  
+
+
+3. Select the appropriate option in the *Extension* drop-down list in the *Settings* section. The following extensions are available for the Cloudinary destination attribute:
+  - **Copy cloudinary image value**: Copy the value from the Cloudinary image attribute and map it to another Cloudinary image attribute.
+  - **Import from cloudinary folder**: Import all image files within a specified folder in Cloudinary to the Cloudinary attribute.    
+
+  The *Configuration* section is displayed below the *Settings* section.
+
+  > [Info] For detailed information about the ETL extensions, see [ETL extensions list](../../DataHub/UserInterface/03_ETLExtensions.md).
+
+4. Select the appropriate tracking mode in the *Override change tracking mode* drop-down list. By default, the **Use default** option is preselected. The following modes are available:
+  - **Use default**: ---
```

Comment:
Vielleicht Beschreibung wie " The default values are applied/used" o.ä. hinzufügen.